File: cfgs/java_cfgs_foo/CHECK_WHETHER_NUMBER_DUCK_NUMBER_NOT.java
Nodes
1;special;;;
2;statement;int len = fun_1();14;
3;statement;int count_zero = 0;15;
4;statement;char ch;16;
5;statement;int i = 1;17;
6;conditional;foo_1 < foo_2;18;
7;statement;ch = fun_1(foo_2);20;
8;conditional;foo_1 == '0';21;
9;statement;count_zero++;21;
10;statement;i++;19;
11;statement;return foo_1;23;
12;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;
5;7;11;
6;8;
7;9;10;
8;10;
9;6;
10;12;
11;

File: cfgs/pythoncfgs/CHECK_WHETHER_NUMBER_DUCK_NUMBER_NOT.py
Nodes
1;statement;foo_1 = fun_1(foo_2);8;
2;statement;foo_1 = 0;9;
3;statement;foo_1 = 1;10;
4;conditional;foo_1 < foo_2;11;
5;statement;return foo_1;16;
6;statement;foo_1 = foo_2[foo_3];12;
7;conditional;foo_1 == '0';13;
8;statement;foo_1 = foo_2 + 1;15;
9;statement;foo_1 = foo_2 + 1;14;
10;exit;;
Edges
0;2;
1;3;
2;4;
3;5;6;
4;10;
5;7;
6;8;9;
7;4;
8;8;
9;
